tests/Unit/Enumerables/JobStatusTest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Tests\Unit;
+
+use EthicalJobs\SDK\Enumerables\JobStatus;
+use Tests\TestCase;
+
+class JobStatusTest extends TestCase
+{
+    /**
+     * @test
+     * @group Unit
+     */
+    public function testFunctionOutputMatchesConst()
+    {
+      $this->assertEquals(JobStatus::DRAFT, JobStatus::DRAFT());
```

Comment:
@bambii7 we need another test (method) to prove `JobStatus::DRAFT()` equals a literal string of 'DRAFT'

This test proves that DRAFT() === ::DRAFT but doesn't prove the value is a literal string of 'DRAFT'

Hope that makes sense

Reply:
@VeryStrongFingers so a test like `$this->assertEquals(JobStatus::DRAFT, 'DRAFT');`

Reply:
@bambii7 Exactly, but `expected` comes first, so:

`$this->assertEquals('DRAFT', JobStatus::DRAFT);`